Solutions/Authomize/Analytic Rules/AWS_role_with_admin_privileges.yaml

```diff
@@ -0,0 +1,61 @@
+id: 734c00a0-a95b-44dd-9b69-d926ed44256d
+name: AWS role with admin privileges
+kind: Scheduled
+description: The policy detect the creation of new AWS roles with administrative privileges.the policy configuration allows limiting the policy to specific accounts.
+severity: High
+queryFrequency: 5m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+- InitialAccess
+relevantTechniques:
+- T1078
+query: |-
+  Authomize_v2_CL
+  | extend EventID = id_s
+  | extend Policy = policy_name_s
+  | extend Severity = severity_s
+  | extend Description = description_s
+  | extend Recommendation = recommendation_s
+  | extend URL = url_s
+  | extend Tactics = tactics_s
+  | where Policy contains "AWS role with admin privileges"
+  | project  EventID, Policy, Severity, Description, Recommendation, URL, Category, Tactics
+requiredDataConnectors:
+- connectorId: Authomize
+  dataTypes: [ "Authomize_v2_CL" ]
+incidentConfiguration:
+  createIncident: true
+  groupingConfiguration:
+    enabled: true
+    reopenClosedIncident: false
+    lookbackDuration: 5h
+    matchingMethod: AnyAlert
+    groupByEntities: []
+    groupByAlertDetails: []
+    groupByCustomDetails: []
+eventGroupingSettings:
+  aggregationKind: SingleAlert
+alertDetailsOverride:
+  alertnameFormat: Alert from Authomize - AWS role with admin privileges
+  alertDescriptionFormat: AWS role with admin privileges - The policy detect the creation of new AWS roles with administrative privileges.the policy configuration allows limiting the policy to specific accounts.
```

Comment:
Just like alertName and alertDescription, please populate alertSeverity and alertTactics.

Reply:
added   
alertSeverity: High
alertTactics: InitialAccess

Reply:
You have to add column names here and not hardcoded values.

Reply:
I will have to investigate - I'm not sure on what you mean here - I'll do some research to see if i can understand - if you have an example please share.

Reply:
OK - I think I undetrstand now - we are saying that we support  the column names from KQL luke under the customDetails section. I have modified the akertSeverity and alertTactics to now reflect a dynamic field.

I think I finally undertstand the dynamic rules - its possible to write one YAML that works for all these Rules as its all the same process.

Got it. OK - For now I'll submit these - but I'm going to modify going forward as you have suggested before so we will eventually have one file with the analytic rules generated dynamically.